Note that `AFT` dates may not indcate an event ocurred (#270)
FamilySearch/GEDCOM: specification/gedcom-03-datamodel.md

    ````
    
    </div>
    
    <div class="note">
    

>     That a `DATE` indicates an event occurred was introduced to in version 5.4 (1995), and is generally followed by applications.

Reviewer: I can't parse this sentence.  Not sure what "to in" means, and also reads awkward starting the sentence with "That".

>     That a `DATE` indicates an event occurred was introduced to in version 5.4 (1995), and is generally followed by applications.

Author: ```suggestion
    The presence of a `DATE` has indicated an event occurred since version 5.4 (1995), and is generally followed by applications.
```

    However, it is common for users to enter a date range with no end without intending to indicate that the event occurred.
    For example, pre 7.0 files sometimes used
    
    ````gedcom
    1 NATU
    2 DATE AFT 1800
    ````
    
    to mean what 7.0 encodes as
    
    ````gedcom
    1 NOT NATU
    2 DATE TO 1800
    ````
    
    without intending to imply that `NATU` ever did actually occur.
    Because this is a "sometimes used" rather than a "formally means" situation,

>     it is likely that data using 5.x "after meaning not before" *de facto* pattern will be transferred as-is into 7.0 and persist in files for the foreseeable future.

Reviewer: Does this suggest that the [migration guide](https://gedcom.io/migrate/) should suggest converting the former (NATU.DATE AFT 1800) to the latter ("NOT NATU.DATE TO 1800")?
